src/app/custom/enrichers/enrichers-catalog.json

```diff
@@ -111,45 +96,85 @@
+        "objectifTDM": "https://services.istex.fr/creation-de-topics-sur-un-corpus-lda/",
+        "type": "classification"
     },
-    {
-        "id": "post-v2-hal-expand",
-        "url": "https://biblio-tools.services.istex.fr/v2/hal/works/expand",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/biblio-tools",
-        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=biblio-tools%20-%20Outils%20pour%20r%C3%A9f%C3%A9rences%20bibliographiques#/biblio-tools/post-v2-hal-works-expand",
-        "objectifTDM": "https://services.istex.fr/2022/02/09/enrichissement-par-doi/",
-        "type": "metadata"
+                        {
+        "id": "post-v1-textClustering",
+        "url": "https://text-clustering.services.istex.fr/v1/clustering",
+        "doc": "https://github.com/Inist-CNRS/web-services/tree/main/services/text-clustering",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=text-clustering%20-%20Cr%C3%A9er%20diff%C3%A9rents%20clusters%20%C3%A0%20partir%20d%27un%20ensemble%20de%20textes%20courts%20ou%20d%27un%20ensemble%20de%20listes%20de%20mots-cl%C3%A9s.#/data-computer/post-v1-clustering",
+        "objectifTDM": "https://services.istex.fr/extraction-de-cluster-dun-corpus/",
+        "type": "classification"
     },
-    {
-        "id": "post-v1-cnrsunit",
-        "url": "https://address-kit.services.istex.fr/v1/cnrsunit/cnrsunit",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/address-kit#v1%2fcnrsunit%2fcnrsunit",
-        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=address-kit%20-%20Informations%20sur%20des%20adresses%20d%27affiliations#/address/post-v1-cnrsunit",
-        "objectifTDM": "https://services.istex.fr/2022/11/09/detection-dunites-cnrs/",
-        "type": "affiliation"
+             {
+        "id": "post-v1-astroTag",
+        "url": "https://astro-ner.services.istex.fr/v1/tagger",
+        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/astro-ner",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=astro-ner%20-%20Cette%20instance%20propose%20un%20outil%20de%20reconnaissance%20d%27entit%C3%A9s%20nomm%C3%A9es%20en%20astronomie#/default/post_v1_tagger",
+        "objectifTDM": "https://services.istex.fr/detection-dentites-nommees-en-astronomie/",
+        "type": "extraction d'entités nommées"
+    } ,   
+              {
+        "id": "post-v1-chemTag",
+        "url": "https://chem-ner.services.istex.fr/v1/chem/tagger",
+        "doc": "https://github.com/Inist-CNRS/web-services/tree/main/services/chem-ner#readme",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=chem-ner%20-%20Trouve%20des%20entit%C3%A9s%20nomm%C3%A9es%20de%20Chimie%20dans%20un%20texte#/default/post_v1_chem_tagger",
+        "objectifTDM": "https://services.istex.fr/detection-dentites-nommees-en-chimie/",
+        "type": "extraction d'entités nommées"
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** Use the key from `translations.tsv`, i.e. `"entites nommees"`

The same for all the services of the same type.‏